package custom_msgs.msgs;

message SimulationParams
{
    required string simulation_folder = 1;
    required string simulation_prefix = 2;
    required string params = 3;
    optional string params_file = 4 [default = ""];
    optional int32 params_line = 5 [default = 0];
    
}